x/simulation/params.go

```diff
@@ -197,7 +197,9 @@ func randomConsensusParams(r *rand.Rand, appState json.RawMessage, cdc codec.JSO
 			MaxAgeDuration:  stakingGenesisState.Params.UnbondingTime,
 		},
 	}
-
+	if maxGas != nil {
```

Comment:
```suggestion
	if maxGas > 0 {
```

Reply:
Yes, but block max gas semantics are that any non-positive value is indicative of being "set". Why not just stick with that?